rules/do-you-involve-stakeholders-in-pi-planning/rule.md

```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
 title: Do you involve business stakeholders in Program Increment (PI) planning?
-uri: do-you-involve-stakeholders-in-pi-planning
+uri: involve-stakeholders-in-pi-planning
```

Comment:
As per my other comments, please update the folder to match the new uri

Reply:
Done ✅ - PR for folder changes: https://github.com/SSWConsulting/SSW.Rules.Content/pull/8357